pact-provider-api/src/main/java/com/example/pactproviderapi/dtos/EligibilityResponse.java

```diff
@@ -8,6 +10,7 @@ public EligibilityResponse(boolean isEligible) {
         this.isEligible = isEligible;
     }
 
+    @JsonProperty("isEligible")
```

Comment:
Without this, the response just contains `eligible`. The contract testing actually caught this for me :D